ansible/hosts

```diff
@@ -1,20 +1,8 @@
-[infinite-staging]
-staging.infinite.industries
+[staging]
+staging.infinite.industries ansible_user=infinite
```

Comment:
I've gone back and fourth on using ip vs using dns for this sort of thing. DNS sounds more flexible but you can't address if there are dns issues and also you shoot yourself in the foot if you pointed the dns wrong. The IP feels more precise. Thoughts?